setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import os
+
+from setuptools import setup, find_packages
+
+setup(
+    name='clcache',
+    description='MSVC Complier cache',
+    author='frerich',
```

Comment:
The [examples](https://docs.python.org/3/distutils/setupscript.html) seem to use the full name here for the author, I think the code should do the same.